### ISSUE TITLE
Fix Software Install Guide - Electrical

### ISSUE THREAD

**Issue by AndrewMcGhee** [oaklandrobotics/docs#7]
Formatting funky and need to add some other programs used for debugging.

**Comment by NerduMiner** [comment]
Could you list the programs that should be added to that document?

**Comment by AndrewMcGhee** [comment]
> Could you list the programs that should be added to that document?

yes. I will do that tomorrow after this report/presentation is turned in 👍

**Comment by AndrewMcGhee** [comment]
> Could you list the programs that should be added to that document?

I believe it's just cangaroo for now. I have a couple other things I want to use for debugging, but I haven't had anyone else try them yet, so I'd like to delay putting them on here if that makes sense.
